Xamarin.PropertyEditing.Mac/PropertyEditorPanel.cs

```diff
@@ -192,8 +192,8 @@ private void Initialize ()
 				NSLayoutConstraint.Create (this.tabStack, NSLayoutAttribute.Bottom, NSLayoutRelation.Equal, this.header, NSLayoutAttribute.Bottom, 1, 0),
 				NSLayoutConstraint.Create (this.tabStack, NSLayoutAttribute.Right, NSLayoutRelation.LessThanOrEqual, this.propertyFilter, NSLayoutAttribute.Left, 1, 0),
 
+				NSLayoutConstraint.Create (this.propertyFilter, NSLayoutAttribute.Left, NSLayoutRelation.Equal, this.tabStack, NSLayoutAttribute.Right, 1, 20),
```

Comment:
We should use Leading and Trailing, instead Left-Right

Reply:
updated

Reply:
Great thanks!